models/job.js

```diff
@@ -141,5 +141,5 @@ module.exports = {
      * @property indexes
      * @type {Array}
      */
-    indexes: [{ fields: ['pipelineId'] }, { fields: ['state'] }]
+    indexes: [{ fields: ['pipelineId', 'state'] }, { fields: ['state'] }]
```

Comment:
` { fields: ['state'] }` do we still need the 2nd field?

Reply:
yes, this is needed, as it's second in the index; this field will not be used when used separately in query;